source/Detector/Amptek/AmptekSDD123Detector.cpp

```diff
@@ -172,7 +177,7 @@ void AmptekSDD123Detector::readStatusData(const QByteArray &statusData){
 
 	backwardsMid(8, 4, statusData, tmpData);
 	internalSetGeneralPurposeCounter(tmpData.toHex().toInt(&ok, 16));
-	//qDebug() << "gpCounter: " << tmpData.toHex() << gpCounter;
```

Comment:
We might want to keep the debugging messages for a little while and clean those up when we are "done"